CONTRIBUTING.md

```diff
@@ -2,6 +2,12 @@
 
 Anyone can contribute to the Kabanero project and we welcome your contributions!
 
+Contribute to Kabanero Open Source communities and earn a badge! With this badge, you earn a right to brag in front of your colleagues and friends! You can also show such badge to your current/future employees that you have been recognized by your contribution to Kabanero, the open-source GitHub communities.
+
+https://www.youracclaim.com/org/ibm/badge/kabanero-contributor
+
+![Kabanero Badge](src/main/content/img/kabanero-badge.jpg)
```

Comment:
I would put this in its own sub header `## Earn the Kabanero Contribution badge`

Reply:
also update it to

>Contribute to the Kabanero Open Source communities and earn a badge! With this badge, you earn a right to brag in front of your colleagues and friends! You can also show this badge to anyone and prove that you have been recognized by your contribution to the open source Kabanero project.